docs/source/release/v6.9.0/inelastic.rst

```diff
@@ -8,36 +8,41 @@ Inelastic Changes
 New Features
 ------------
 - The :ref:`Interface Settings <interface-indirect-settings>` has a new advanced option called ``Developer Feature Flags``. When developing a large new feature, developers will create a flag that can be used by advanced software users for purposes of rapid prototyping and allowing a faster feedback loop.
```

Comment:
Good spot, yes you are right that both Indirect and Inelastic interface categories share a common settings widget. This link for the settings page is however a bit outdated because it suggests that the settings widget is only used by `indirect`. I have updated the link to a better name `inelastic-interface-settings`. This is a better description because inelastic encompasses indirect, whereas indirect does not encompass all of inelastic (we can also have direct inelastic)!